src/qml/imports/Theme/QfToolButton.qml

```diff
@@ -110,5 +113,25 @@ Item {
 
             }
         }
+
+        Rectangle {
+            id: bottomRightIndicator
+            color: bottomRightIndicatorBgColor
+            anchors.bottom: button.bottom
+            anchors.right: button.right
+            width: button.width / 3
+            height: width
+            visible: bottomRightIndicatorText
+            radius: width
```

Comment:
```suggestion
            radius: width
            border.width: 1.5
            border.color: 'white'

```
Would that look good?

Reply:
![image](https://user-images.githubusercontent.com/2820439/120766108-91c37d80-c522-11eb-98db-6637e3055ab0.png)
not convinced

Reply:
![image](https://user-images.githubusercontent.com/2820439/120766247-b61f5a00-c522-11eb-8277-051b33b4b23e.png)
a bit better but I like it more without the border

Reply:
![image](https://user-images.githubusercontent.com/2820439/120766382-dc44fa00-c522-11eb-953a-9c5056c91cfd.png)

Reply:
@suricactus , what about 1/ making the text a tiny bit smaller, 2/ moving it up/left so it overlaps with the cloud icon a bit more (ATM it's pretty close to the bottom edge of the rect), and 3/ add a green background border.

Reply:
1) I would like to add some padding, so I do agree
2) I would like to keep it down there for now, since these are the "local" changes. The topright conrder might be "cloud changes" one day.  If we also show the remote changes one day, it would be confusing to move it around. If we are certain we don't, or use another approach, then I am ok to move it to the top.
3) I am using the same green background as the avatar image:
![image](https://user-images.githubusercontent.com/2820439/120769693-0ba93600-c526-11eb-8025-ef04d4add888.png)

Reply:
What happens if we have more than 10 pending changes?
Should we just show a notification icon that things are pending without numbers?

- one single notification dot (for local and cloud)
- overlapping the cloud
- in a different color
- with a green "buffer" circle

Reply:
@suricactus , right, I think bottom left is fine, I was suggesting to move it up a little bit, like this:
![image](https://user-images.githubusercontent.com/1728657/120771000-f396de00-c548-11eb-91c9-4334ce286eef.png)

Reply:
@m-kuhn , re >= 10, if we want to keep some numbers, we could go for 1,2,3,4,5,6,7,8,9, and for 10 and above, we just use the '+' sign.

Reply:
Let's go with that one !

Reply:
![image](https://user-images.githubusercontent.com/2820439/120775437-def81d00-c52b-11eb-81a2-a1b5a90ba172.png)
![image](https://user-images.githubusercontent.com/2820439/120775587-02bb6300-c52c-11eb-9596-fddab82527a9.png)

I think this the font size is on the edge of readability and we cannot make it as nice as Mathieu's screenshot, which requires a bit smaller circle.

Reply:
@suricactus , oh, I really like this! 

For >= 10, I'd just use a + sign, no '9+'.

Reply:
![image](https://user-images.githubusercontent.com/2820439/120776452-f1268b00-c52c-11eb-90a7-8603c155affa.png)
not so sure

Reply:
I love it, @m-kuhn , thoughts?

Reply:
That works. Or we kill the number and just use a different color for the bullet (less noise and very clear to the user that "something needs my attention")

Reply:
my original idea was to use color only. Then I thought I am not good at colors, so I am open for suggestion.